src/bvals/cc/bvals_cc.hpp

```diff
@@ -75,7 +75,7 @@ class CellCenteredBoundaryVariable : public BoundaryVariable {
   bool ReceiveFluxCorrection(bool is_allocated) final;
 
  protected:
-  int nl_, nu_;
+  int ll_, lu_, ml_, mu_, nl_, nu_;
```

Comment:
It looks like above the size of each dimension is assumed to be `nu_ + 1`, etc. Is `nl_`, etc. supposed to note the lower bound of each dimension? Should the sizes be `nu_ - nl_ + 1` or is something else going on? I assume things are correct as they are now, but it would be good to have a comment here about what these variables denote.

Reply:
Good point -- `nsize = nu_ - nl_` wasn't originally used in `flux_correction` but now it is, and it should be `nu_ - nl_ + 1` like you say.

Reply:
It looks like it is still `nsize = nu_ - nl_`, etc. in flux_correction_cc.cpp.

Reply:
Looks good now, not sure what was going on before.